### PR TITLE
GUACAMOLE-1813: Update Guacamole Docker documentation to include enabling recording extension.

### PR DESCRIPTION
**Pull request by necouchman**

### CODE REVIEW

File: src/guacamole-docker.md [hunk 1]
```diff
@@ -782,6 +786,36 @@ SAML authentication extension.
...
+
+The extension that enables viewing historical recordings from within the
+Guacamole Client interface can be enabled by settings the search path
+variable, as noted below, to a location where the extension will look
+to find available recordings.
+
+When setting this up in a container environment, you'll likely need
+to use volumes to make the same directory available to both the
+guacd container and the guacamole (client) container. Setting up these
+volumes is beyond the scope of this document.
```
Comment (mike-jumper) [inline src/guacamole-docker.md:801]: Should we include the UIDs/GIDs used by the Guacamole and guacd images? Could be good to provide known-good values.

Comment (necouchman) [reply src/guacamole-docker.md:801]: Sure, happy to add those values in here.

File: src/guacamole-docker.md [hunk 2]
```diff
@@ -782,6 +786,42 @@ SAML authentication extension.
...
+to find available recordings.
+
+When setting this up in a container environment, you'll likely need
+to use volumes to make the same directory available to both the
+guacd container and the guacamole (client) container. In addition
+to setting up the volume to share data between the two, you'll also
+need to configure permissions on the volume such that the users
+running each of the containers have access. The guacd container,
+which will need write access to this shared location, runs with the
+effective UID and GID of 1000. The guacamole-client container, which
```
Comment (mike-jumper) [inline src/guacamole-docker.md:805]: I recommend referring to these by their Docker Hub image names (`guacamole/guacd` and `guacamole/guacamole` respectively).

**Review by mike-jumper — CHANGES_REQUESTED** [review]

File: src/guacamole-docker.md (hunk 2 above)
Comment (necouchman) [reply src/guacamole-docker.md]: Ah, yes...fixed via rebase.

**Comment by necouchman** [comment]
@mike-jumper Any other updates required, here?

**Review by mike-jumper — APPROVED** [review]
Looks good to me!

**Pull request merged**